garage/tf/models/base.py

```diff
@@ -188,8 +188,7 @@ def build(self, *inputs, name=None):
         if not self._networks:
             # First time building the model, so self._networks are empty
             # We store the variable_scope to reenter later when we reuse it
-            with tf.variable_scope(
-                    self._name, reuse=False) as self._variable_scope:
+            with tf.variable_scope(self._name) as self._variable_scope:
```

Comment:
please explicitly assign the new a scope into a member variable -- do not capture it as a coincidence of a context manager

Reply:
Forgot to modify it here. Will fix.

Reply:
bump